### ISSUE TITLE
[meta] Duplication with isaacs/github

### ISSUE THREAD

**Issue by jayvdb** [dear-github/dear-github#207]
There is a lot of overlap in these issues with https://github.com/isaacs/github/issues .  Is it intentional/ desirable? Should issues be cross-linked in the two repos?

Adding a `CONTRIBUTING.md` would be helpful to indicate whether this repo is desiring new issues, and how they should be filed. c.f. https://github.com/isaacs/github/blob/master/CONTRIBUTING.md

Perhaps creating a `.github/ISSUE_TEMPLATE.md` in this repo would be good, to indicate to search there first, and also shows of the new functionality added in https://github.com/blog/2111-issue-and-pull-request-templates .

**Comment by axd1967** [comment]
* https://github.com/isaacs/github/issues/551#issuecomment-268756918
* as a result, some users duplicate stuff: https://github.com/isaacs/github/issues/822 ...

**Comment by cirosantilli** [comment]
Proposal: close the issues here, and on the README clearly redirect people to Isaacs.

Migrate all existing issues there.

Rationale:

1) more issues there, so migration is easier.
2) this is designed as a petition to be signed. It has been signed, and is hard to change now. But Isaacs has been a GitHub forum from beginning.

Counter rationale: this has more stars, because it got more flashy Reddit coverage :-)

Finally, both will die soon when GitHub implements the promised public forum.

**Comment by crutchcorn** [comment]
@cirosantilli Might be nice to have a way to migrate everything over anyway. I can help take items and pop them into the public forum when it goes public if needed

**Comment by aspiers** [comment]
I've submitted #295 as a partial fix for this.  The other thing needed for a full fix would be a PR auto-closer bot as mentioned in https://github.com/isaacs/github/issues/1191.

**Comment by aspiers** [comment]
See also [#54 (Link to isaacs/github)](https://github.com/dear-github/dear-github/issues/54).

**Comment by axd1967** [comment]
A superficial comment: I'm beginning to wonder if this has become a positional war, where neither party is willing to give up.

A bit related to the many Facebook <ahem> groups that pop up around common causes, but actually create islands that prevent a single, coordinated front. 

Divide and reign, something like that: a nasty side effect of the contemporary social media?

**Comment by oprogramador** [comment]
If the https://github.com/dear-github/dear-github maintainers do nothing, IMO we can do the following:
- migrate all the issues from https://github.com/dear-github/dear-github into https://github.com/isaacs/github/issues
- try to get more stars for https://github.com/isaacs/github

**Comment by cirosantilli** [comment]
Issue migration would be relatively easy (300 * 3 clicks or so) with the new issue migration feature: https://help.github.com/en/articles/transferring-an-issue-to-another-repository

However I've tried and you cannot transfer across orgs / users it seems...  so feature request: https://github.com/isaacs/github/issues/1604

I wouldn't do anything to the issues here before that becomes possible.

**Comment by jayvdb** [comment]
There are lots of issues in the target repository which are the same as the ones here.

It wouldnt be helpful to create duplicates there.

We would need a list of issues here which are not present in the other repo, if any.

**Comment by markcellus** [comment]
https://github.com/isaacs/github/issues is read-only now. Can we close?

**Comment by aspiers** [comment]
Unfortunately GitHub decided to handle https://github.com/isaacs/github/issues in the worst possible way, [just like I feared but hoped wouldn't happen](https://github.com/isaacs/github/issues/2035#issuecomment-920887705).

So now we have literally thousands of issues and probably 100s of thousands of person-hours of collective wisdom about GitHub gathered over ~10 years which is now all locked in a read-only repository and partially duplicated not only in this repo but in the new(-ish) GitHub Discussions official forums.  I hope the official forums provide an adequate outlet for community discussions going forwards and forever more, but if not we'll be back to square one of needing to rely on an unofficial outlet like this one.

**Comment by TPS** [comment]
My 2¢ as a former final-days collaborator (think 'moderator') @ `isaacs/github`:

> partially duplicated not only in this repo but in the new(-ish) GitHub Discussions official forums.  I hope the official forums provide an adequate outlet for community discussions going forwards and forever more, but if not we'll be back to square one of needing to rely on an unofficial outlet like this one.

For the record, the new official fora are @  https://github.com/orgs/community/discussions/, & my cautious optimism in deprecating `isaacs/github` remains exactly @ the same level as @aspiers' since the official's day 1.  There has been incremental, ever-more-glacial progress to the point that not only is a presence for many both here & there is necessary, but also (for some) in more factioned groups e.g., https://github.com/drop-ice/dear-github-2.0, [those w/ the concerns of code theft via AI use](https://github.com/orgs/community/discussions/20284), the [continuing ramifications 4½ years on of the slow absorption by Microsoft](https://techcrunch.com/2022/10/26/four-years-after-being-acquired-by-microsoft-github-keeps-doing-its-thing/), &c.

TLDR: I believe not only that this repo, but this _issue_ (w/ title modified to point to official GitHub discussions as the successor to `isaacs/github`) should stay open, & periodically reevaluated to determine whether our voices are effective as "just" users (as opposed to management in either corporation or stockholders thereof).
